projects/vdk-core/src/vdk/internal/builtin_plugins/config/log_config.py

```diff
@@ -57,12 +64,21 @@ def configure_loggers(
         "stream": "ext://sys.stderr",
     }
 
+    _SYSLOG_HANDLER = {
+        "level": "DEBUG",
+        "class": "logging.handlers.SysLogHandler",
+        "formatter": "detailedFormatter",
+        "address": log_endpoint,
+        "socktype": socket.SOCK_DGRAM,
+        "facility": "user",
+    }
+
     if "CLOUD" == log_config_type:
         CLOUD = {  # @UnusedVariable
             "version": 1,
-            "handlers": {"consoleHandler": _CONSOLE_HANDLER},
+            "handlers": {"consoleHandler": _CONSOLE_HANDLER, "SysLog": _SYSLOG_HANDLER},
             "formatters": _FORMATTERS,
-            "root": {"handlers": ["consoleHandler"]},
+            "root": {"handlers": ["consoleHandler", "SysLog"]},
```

Comment:
This implies that the Syslog handler is always enabled. But it should not be. Not everyone would need it.

Reply:
included a SYSLOG_ENABLED config var which makes it so the syslog handler is only included if the config var is set to True